app/src/test/java/nerd/tuxmobil/fahrplan/congress/sharing/JsonLectureFormatTest.kt

```diff
@@ -0,0 +1,35 @@
+package nerd.tuxmobil.fahrplan.congress.sharing
+
+import com.google.common.truth.Truth.assertThat
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+import org.junit.Test
+
+class JsonLectureFormatTest {
+
+	@Test
+	fun formatSingle() {
+		val lecture = Lecture("lecture1")
+		val json = JsonLectureFormat.format(lecture)
+		println(json)
+		assertThat(json).isEqualTo(EXPECTED_JSON_SINGLE)
+	}
+
+	@Test
+	fun formatList() {
+		val lectures = listOf(
+				Lecture("lecture1"),
+				Lecture("lecture2"),
+				Lecture("lecture3"))
+		val json = JsonLectureFormat.format(lectures)
+		println(json)
+		assertThat(json).isEqualTo(EXPECTED_JSON_LIST)
+	}
+
+	companion object {
+
+		private const val EXPECTED_JSON_SINGLE = "{\"lectures\":[{\"lectureId\":\"lecture1\",\"title\":\"\",\"subtitle\":\"\",\"day\":0,\"room\":\"\",\"slug\":\"\",\"speakers\":\"\",\"track\":\"\",\"type\":\"\",\"lang\":\"\",\"abstract\":\"\",\"description\":\"\",\"links\":\"\"}]}"
```

Comment:
# JSON format
Don't you want to configure Moshi in `JsonLectureFormat` and in the  to convert camel-case to snake-case in order to have a more JSON-like output format, e.g. `lecture_id` instead of `lectureId`?

Reply:
Looks like [Moshi doesn't recommends](https://publicobject.com/2016/01/20/strict-naming-conventions-are-a-liability/) case-conversion. Using an annotation in `LectureExport` looks fine to me.